### PR TITLE
add in custom configmap for containerinsights

### PR DESCRIPTION
**Pull request by cakeben**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by zolibh — APPROVED** [review]

File: k8s/namespaces/kube-system/container-axm-ms-agentconfig/container-azm-ms-agentconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  log-data-collection-settings: |-
+    # Log data collection settings
+    [log_collection_settings]
+       [log_collection_settings.stdout]
+          # In the absense of this configmap, default value for enabled is true
+          enabled = true
+          # exclude_namespaces setting holds good only if enabled is set to true
+          # kube-system log collection is disabled by default in the absence of 'log_collection_settings.stdout' setting. If you want to enable kube-system, remove it from the following setting.
+          # If you want to continue to disable kube-system log collection keep this namespace in the following setting and add any other namespace you want to disable log collection to the array.
+          # In the absense of this configmap, default value for exclude_namespaces = ["kube-system"]
```
Comment (adusumillipraveen) [inline k8s/namespaces/kube-system/container-axm-ms-agentconfig/container-azm-ms-agentconfig.yaml:19]: Don't you need to do it her as well ?

**Review by adusumillipraveen — COMMENTED** [review]
Adding this file wont do anything in flux. You need a file like this https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/admin/aad-pod-id/kustomization.yaml

and referred in each environment like https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/cluster-00-overlay/kustomization.yaml#L5

easiest way is going to be putting it in https://github.com/hmcts/cnp-flux-config/tree/master/k8s/common, but this will make it very difficult to change / config per env in future.

**Comment by adusumillipraveen** [comment]
Also, u need to remove this file https://github.com/hmcts/cnp-flux-config/blob/master/k8s/sandbox/common/container-azm-ms-agentconfig.yaml , it will cause conflict.

File: k8s/demo/cluster-00-overlay/kustomization.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
+§900o§§§§§apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (timja) [inline k8s/demo/cluster-00-overlay/kustomization.yaml:1]: :shipit:

Comment (cakeben) [reply k8s/demo/cluster-00-overlay/kustomization.yaml]: I saw this first

Comment (timja) [reply k8s/demo/cluster-00-overlay/kustomization.yaml]: you guys are talking in code

**Review by timja — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**